### PR TITLE
Gatsby Sitemap Pluggin

### LINKED ISSUE #522: [NG] Add sitemap plugin
**Desired Behavior**
Add and configure the [gatsby plugin for sitemap](https://www.gatsbyjs.org/packages/gatsby-plugin-sitemap/?=sitemap) in [layer5-ng](https://github.com/layer5io/layer5/tree/layer5-ng). Related to #481

### PR DESCRIPTION
**Pull request by PluckyPrecious**
Signed-off-by: PluckyPrecious <prekel27@gmail.com>

**Description**

This PR fixes 

**Notes for Reviewers**
@Tanuj22 kindly check it out, hope it gets fixed. 😄 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -54,3 +54,10 @@ module.exports = {
     },
   ],
 }
+module.exports = {
```
Comment (Tanuj22) [inline gatsby-config.js:57]: You don't need to add another `module.exports`. You can simply the contents you would like to add to the `module.exports` already present.

Comment (PluckyPrecious) [reply gatsby-config.js:57]: Oh I see! Let me rectify it.

**Comment by PluckyPrecious** [comment]
> @PluckyPrecious Looks like the build is failing here also so I think it was not an issue with your system.
> You also need to run npm install --save gatsby-plugin-sitemap to install the plugin.

I already did install the pluggin

**Comment by Tanuj22** [comment]
> > @PluckyPrecious Looks like the build is failing here also so I think it was not an issue with your system.
> > You also need to run npm install --save gatsby-plugin-sitemap to install the plugin.
> 
> I already installed the pluggin

That's strange. If you had installed it should have been added to `package.json` automatically. That's what causing the build to fail

**Comment by PluckyPrecious** [comment]
> @PluckyPrecious Looks like the build is failing here also so I think it was not an issue with your system.
> You also need to run npm install --save gatsby-plugin-sitemap to install the plugin.
> 
> I already installed the pluggin
> 
> That's strange. If you had installed it should have been added to package.json automatically. That's what causing the build to fail

This what i get when i run build

```

> Layer5@1.0.0 build C:\Users\HP\Layer5
> gatsby build

'gatsby' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! Layer5@1.0.0 build: `gatsby build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the Layer5@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

```

**Comment by Tanuj22** [comment]
@PluckyPrecious did you run `make setup-libs` before starting the project?

**Comment by PluckyPrecious** [comment]
> @PluckyPrecious did you run make setup-libs before starting the project?

No i didn't, okay then let me redo it .

**Comment by Tanuj22** [comment]
> > @PluckyPrecious did you run make setup-libs before starting the project?
> 
> No i didn't, okay then let me redo it .

You can follow [these steps](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md#building-the-site) to check the site locally :)

**Pull request closed without merging**